Gruntfile.js

```diff
@@ -97,7 +97,7 @@ module.exports = function (grunt) {
           // jquery.qtip.css -> _jquery.qtip.scss
           {
             expand: true,
-            cwd: 'node_modules/@okta/qtip2/dist/',
+            cwd: 'packages/@okta/qtip2/dist/',
```

Comment:
It's a good question - the general purpose of retire.js is to see if we have dependencies that have hit a vulnerability list. This covers both the dependencies we're pulling down (in node_modules), as well as anything in the packages folder. You're right - I need to add packages to this list (will update now).